backend/internal/adapters/secondary/repositories/user.go

```diff
@@ -53,7 +53,7 @@ func (r *userRepository) GetUsersByIds(ids []int64) (map[int64]users.User, error
 }
 
 func (r *userRepository) GetByUsername(username string) (*users.User, error) {
-	sql, _, _ := qb.From("users").Where(goqu.C("username").Eq(username)).ToSQL()
+	sql, _, _ := qb.From(goqu.T("users")).Where(goqu.C("username").Eq(username)).ToSQL()
```

Comment:
:o is this necessary? I think some of the other stuff i wrote didn't have this

Reply:
Shouldn't be, but just in case HAHA